docs/workflow.rst

```diff
@@ -3,255 +3,182 @@
 
-Note of warning: if you are using this wrapper for multiple fields,
-check that the proper version of datamodel.py is in place before
-running the batch trimming/fitting scripts.  For instance, if you have
-recently used the wrapper to do part of the processing for field_A,
-and you want to start the batch fitting script for field_B, re-run the
-wrapper for field_B to make sure that datamodel.py refers to the
-information for field_B.
+  * Option 1 (datamodel.ast_source_density_table is set):
+    For each source density or background bin, randomly place the SEDs
+    within pixels of that bin.  Repeat for each of the bins.
+  * Option 2 (datamodel.ast_source_density_table = None):
+    Randomly choose a star from the photometry catalog, and place the
+    artificial star nearby.  Repeat until all SEDs have been placed.
 
+.. code-block:: console
 
-****
-Data
-****
+   $ beast.tools.run.make_ast_inputs --flux_bin_method=True
 
-The data need to have source density information added as it is common
-for the observation model (scatter and bias) to be strongly dependent
-on source density due to crowding/confusion noise.  The background may
-also be important in some situations, so there is code to calculate it as well.
 
-Adding source density or background to observations
-===================================================
+These ASTs should be processed with the same code that was used to extract the
+source photometry.
 
-Create a new version of the observations that includes a column with the
-source density.  The new observation file includes only sources that have
-measurements in all bands (columns that match 'X_RATE').  In theory, sources
-without measurements in all bands is the result of non-overlapping observations.
-The BEAST is based on fitting sources with the same selection function,
-in this case measurements in all bands.
 
-A number of source density images are also created.  These include images
-that map the source density of objects with zero fluxes in different bands
-(or any band).
+*******************
+Edit/Split Catalogs
+*******************
 
-Command to create the observed catalog with source density column with
-a pixel scale of 5 arcsec using the 'datafile.fits' catalog.
+You may wish to remove artifacts from the photometry catalog.  If you do so, the
+same criteria must be applied to the AST catalog.
 
-  .. code-block:: console
+Commands to edit the files, both to remove flagged sources and eliminate sources
+that don't have full imaging coverage:
 
-     $ ./beast/tools/create_background_density_map.py sourceden -catfile datafile.fits --pixsize 5.
+.. code-block:: console
+
+   $ beast.tools.cut_catalogs phot_catalog_with_sourceden.fits phot_catalog_cut.fits \
+         --partial_overlap --region_file --flagged --flag_filter F475W
+   $ beast.tools.cut_catalogs ast_catalog.fits ast_catalog_cut.fits \
+         --partial_overlap --region_file --flagged --flag_filter F475W
```

Comment:
Need to covert commands to command line versions (see prev comments).

Reply:
resolved per discussion below